packages/foam-vscode/src/features/navigation-provider.spec.ts

```diff
@@ -232,6 +232,5 @@ describe('Document navigation', () => {
         range: new vscode.Range(0, 23, 0, 23 + 9),
       });
     });
-    it('should provide references for placeholders', async () => {});
```

Comment:
I would still keep the test case (not sure why it wasn't implemented), maybe `it.todo`?

For me a test case serves 2 purposes:
- tests that the code is actually working
- provides documentation to (present and) future developers about the intended specs of a feature

Sometimes even if a test is convoluted to implement, it might still be worth keeping it as some sort of documentation, although I think those would need to be flagged properly.
I still consider this an edge case (again, not sure what happened here), but an edge case we should account for.

What do you think?

Reply:
I'm going to comment it so that we get a warning.